sunpy/coordinates/frames.py

```diff
@@ -847,3 +856,99 @@ class GeocentricEarthEquatorial(SunPyBaseCoordinateFrame):
     Aberration due to Earth motion is not included.
     """
     equinox = TimeFrameAttributeSunPy(default=_J2000)
+
+
+class BaseMagnetic(SunPyBaseCoordinateFrame):
+    """
+    Base class for frames that rely on the Earth's magnetic model (MAG, SM, and GSM).
+
+    This class is not intended to be used directly and has no transformations defined.
+    """
+    magnetic_model = Attribute(default='igrf13')
```

Comment:
Does the "igrf13" model get updated every 5 years? So the file at the URL below just gets changed?

Reply:
No, the file `igrf13coeffs.txt` should never change.  In ~2025, there will be a newly available `igrf14coeffs.txt`.

Reply:
So what would be our update plan? Change the default model name in a new feature release of sunpy?

Given this I would think the best idea would be to have a way of first checking if we have included the file and then trying to fetch it from online if not. This would let people use newer ones their version does not support as well as older ones we might not have included if they are trying to reproduce some analysis.

I tested to see if you can pass a `file://` url to our data manager / cache but unfortunately not. I think that would be my preferred solution long term (get them from the global cache with the first mirror being a `file://`).

Reply:
> So what would be our update plan? Change the default model name in a new feature release of sunpy?

Yeah

> Given this I would think the best idea would be to have a way of first checking if we have included the file and then trying to fetch it from online if not.

Now done, and I've bundled the IGRF-13 coefficients, so no downloading is needed for the default model